jgiven-html5-report/src/main/java/com/tngtech/jgiven/report/html5/Html5AttachmentGenerator.java

```diff
@@ -105,6 +112,9 @@ private File writeFile( AttachmentModel attachment, MediaType mediaType ) {
         File targetFile = getTargetFile( attachment.getFileName(), extension );
         try {
             if( attachment.isBinary() ) {
```

Comment:
I think that is not enough. The attachment could be binary, but no image (e.g. a ZIP file).